src/supporting-data/diseases/config/DiseasesColumnConfiguration.tsx

```diff
@@ -58,24 +62,41 @@ DiseasesColumnConfiguration.set(DiseasesColumn.alternativeNames, {
     ) : null,
 });
 
-// NOTE: should probably be links
 DiseasesColumnConfiguration.set(DiseasesColumn.crossReferences, {
   label: 'Cross references',
-  // TODO: https://www.ebi.ac.uk/panda/jira/browse/TRM-25838
   render: ({ crossReferences }) =>
-    crossReferences?.length ? (
+    Boolean(crossReferences?.length) && (
```

Comment:
I believe ExpandableList now doesn't render anything if it is passed either undefined, or an empty list, so I would say that this check was not even needed in the first place, and you'd be able to just remove it altogether